app/views/layouts/video_background.html.erb

```diff
@@ -0,0 +1,4 @@
+<%# TODO: Use local version of this video.%>
+<video autoplay loop id="bgvid">
+    <source src="https://www.pexels.com/video/1437396/download" type="video/mp4">
+</video>

```

Comment:
This video is looking grainy on my computer.  It also takes a long time to load.  I think we should just do a still image

Reply:
Fixed with be64470